### DISCUSSION TITLE
Is there any way to use output file using bash to be available as input to build.xml?

### DISCUSSION THREAD

**Discussion by nitssnatha** [community/community#26950] [Actions]
<p>I am running the below command; </p>
<p>steps:<br>
    - uses: actions/checkout@v1<br>
    - name: Get Git Id and diff files<br>
      run: |<br>
        echo “This is the current commit” $GITHUB_SHA<br>
        sha_pre=$(git log --pretty=format:"%H" --merges -n 1)<br>
        echo “This is the previous commit” $sha_pre<br>
        sha_diff=$(git diff --pretty=format: --name-only $GITHUB_SHA $sha_pre)<br>
        echo “$sha_diff” &gt; DownloadDiff.txt<br>
        cat -n DownloadDiff.txt</p>
<p>I want this Diff.txt file as an input for Build.xml in the same project. I have a command "</p>
<p>&lt;loadfile property=“textFile” srcfile="./DownloadDiff.txt" /&gt; in build.xml to fetch the file. But unfortunately its not working.<br>
Below is the output <br>
“/home/runner/work/SalesforceDXCICD/SalesforceDXCICD/build.xml:10: /home/runner/work/SalesforceDXCICD/SalesforceDXCICD/DownloadDiff.txt doesn’t exist”</p>
<p>Please Help"</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/nitssnatha">@nitssnatha</a> ,</p>
<p>Whether the build step is in the same job with “checkout” and “Get Git Id and diff files”?</p>
<p>If not in the same job, you need to use <a href="https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts" rel="nofollow noopener"><strong>artifacts</strong></a> or <a href="https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows" rel="nofollow noopener"><strong>cache</strong></a> to share the files between jobs or workflow runs.</p>
<p>If the problem still exists, please share your repository with us, so that we can check more detailed configurations of the workflow and the build settings in your repository to analyze the root cause.</p>

**Comment by nitssnatha** [comment]
<p>I have used the upload and download artifact approach. But it seems like taking much space.</p>
